### PR TITLE
Fix cpr sideckick

### PR DESCRIPTION
**Pull request by JuliaFernee**

### CODE REVIEW

**Comment by catalin-ilea** [comment]
lgtm

**Comment by tosan88** [comment]
:+1:

File: content-public-read-sidekick@.service [hunk 1]
```diff
@@ -10,7 +10,7 @@ ExecStart=/bin/sh -c '\
   etcdctl mkdir /ft/services/$SERVICE/servers; \
   etcdctl set 	/ft/healthcheck/$SERVICE-%i/path 		/__health; \
   etcdctl set 	/ft/healthcheck/$SERVICE-%i/categories 	read; \
-  etcdctl set 	/vulcand/frontends/public-services-content/frontend  \'{"Type": "http", "BackendId": "vcb-content-public-read", "Route": "PathRegexp(`/content/[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}.*`) && Host(`public-services`)"}\'; \
+  etcdctl set 	/vulcand/frontends/public-services-content/frontend  \'{"Type": "http", "BackendId": "vcb-content-public-read", "Route": "PathRegexp(`/content/.*`) && Host(`public-services`)"}\'; \
```
Comment (tosan88) [inline content-public-read-sidekick@.service]: isn't this change now contradicting with this? https://github.com/Financial-Times/up-service-files/blob/master/public-content-by-concept-api-sidekick%40.service#L12

Comment (Zapadlo) [inline content-public-read-sidekick@.service]: I *think* that not used: http://git.svc.ft.com/projects/CP/repos/api-component-policy-varnish/browse/default.vcl#40 and should be removed.

**Comment by Zapadlo** [comment]
@JuliaFernee Could you double check with @nwrigh and remove this https://github.com/Financial-Times/up-service-files/blob/master/public-content-by-concept-api-sidekick%40.service#L12 as part of the PR?

**Comment by khatton-ft** [comment]
:+1:

**Pull request merged**